### PR TITLE
Create main.cpp

### PR DESCRIPTION
**Pull request by usaotearoa**
Added another example of combined publisher and subscriber. Works as far as I can tell.

### CODE REVIEW

**Comment by usaotearoa** [comment]
Example of combined publisher and subscriber using single node

File: examples/micro-ros_pub_sub/src/main.cpp [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+// publisher and subscriber common
+rcl_node_t node;
+rclc_support_t support;
+rcl_allocator_t allocator;
+rclc_executor_t executor;
+rcl_timer_t timer;
+unsigned int num_handles = 1 + 1;   // 1 subscriber, 1 publisher
+
+#define LED_PIN 13
```
Comment (Acuadros95) [inline examples/micro-ros_pub_sub/src/main.cpp:29]: Examples should be as generic as posible, please leave out specific leds. 
You can use the [micro-ros_publisher example](https://github.com/micro-ROS/micro_ros_platformio/blob/main/examples/micro-ros_publisher/src/Main.cpp) as reference

Comment (usaotearoa) [reply examples/micro-ros_pub_sub/src/main.cpp:29]: Actually, the LED_PIN is used in the original Arduino example. So it has a right to stay in, as this is a direct port from Arduini micro-ros examples.
I'm closing this pull-request nevertheless and will create a new one, sanitized with appropriate directory structure and all file, such as platformio.ini included.

**Review by usaotearoa — COMMENTED** [review]
This is appropriate as per Original Template Example from Arduino. No need to change anything

**Comment by usaotearoa** [comment]
Closing this and opening a new version with platformio.ini and appropriate directory structure

**Pull request closed without merging**